Add EBS volume gp3 post (#2403)
18F/cg-site: _posts/2023-9-7-EBS-storage-update.md

---
layout: post

> title: EBS volumes upgraded

Reviewer: ```suggestion
title: Brokered EBS volumes and storage upgraded to gp3
```

date: September 7, 2023

> excerpt: "All EBS volumes across brokered plans have been upgraded to gp3"

Reviewer: ```suggestion
excerpt: "EBS volumes and storage across all brokered plans have been upgraded to gp3"
```

---

> # EBS volumes increased to gp3

Reviewer: ```suggestion
# EBS volumes and storage upgraded to gp3
```

> Cloud.gov are happy to announce that EBS SSD volumes across all of our brokered plans have been upgraded to high-performance gp3 type. The gp3 type comes with an increased max throughput per volume/instance, a full breakdown comparison of gp3 vs gp2 can be found [here] (https://aws.amazon.com/ebs/general-purpose/)

Reviewer: ```suggestion
Cloud.gov is happy to announce that we have upgraded all our brokered Elasticsearch/Opensearch plans from `gp2` volumes to the high-performance `gp3` volumes by default. We have also upgraded all of our brokered database plans to use `gp3` as the storage volume type by default.

The `gp3` type comes with an increased max throughput per volume/instance versus the previous `gp2` type. A full breakdown comparison of `gp3` vs `gp2` can be found [here] (https://aws.amazon.com/ebs/general-purpose/).
```

Updated the wording here to make it clear how the changes differ per resource type (database vs elasticsearch/opensearch)

---
layout: post
title: Brokered EBS volumes and storage upgraded to gp3
date: September 7, 2023
excerpt: "All EBS volumes across brokered plans have been upgraded to gp3"
---

> # EBS volumes and storage upgraded to gp3

Reviewer: ```suggestion
## EBS volumes and storage upgraded to gp3
```

Cloud.gov is happy to announce that we have upgraded all our brokered Elasticsearch/Opensearch plans from `gp2` volumes to the high-performance `gp3` volumes by default. We have also upgraded all of our brokered database plans to use `gp3` as the storage volume type by default.

> The `gp3` type comes with an increased max throughput per volume/instance versus the previous `gp2` type. A full breakdown comparison of `gp3` vs `gp2` can be found [here] (https://aws.amazon.com/ebs/general-purpose/).

Reviewer: ```suggestion
The `gp3` type comes with an increased max throughput per volume/instance versus the previous `gp2` type. A full breakdown comparison of `gp3` vs `gp2` [can be found here](https://aws.amazon.com/ebs/general-purpose/).
```

Link wasn't rendering properly - this will fix it

> # What do you need to do?
> 
> In most cases, there is no immediate action required on your part. You're existing EBS volumes have been automatically updated to gp3 for your convenience. 

Reviewer: ```suggestion
## How does this affect me?

These changes will only be applied to **new services (database, Elasticsearch/Opensearch)** that are created. Existing service instances will continue using `gp2` volumes.
```

This is an important clarification. By updating the broker plans, we've changed the defaults that new services are created with. But existing services will be unaffected.
